pkg/controller/windowsmachineconfig/nodeconfig/nodeconfig.go

```diff
@@ -45,13 +49,26 @@ var log = logf.Log.WithName("nodeconfig")
 
 // Configure configures the Windows VM to make it a Windows worker node
 func (nc *nodeConfig) Configure() error {
+	var err error
+	// Validate WindowsVM
+	if err := nc.Windows.Validate(); err != nil {
+		return errors.Wrap(err, "Unable to validate Windows VM")
```

Comment:
`error validating Windows VM`

The pattern is to not start messages with capitals.